packages/compass-schema-validation/src/modules/validation.ts

```diff
@@ -484,3 +485,9 @@ export const activateValidation = (): SchemaValidationThunkAction<void> => {
     void dispatch(fetchValidation(namespace));
   };
 };
+
+export const hasErrorAndLogValidationActionSupport = (
+  serverVersion: string
+) => {
+  return semver.gte(serverVersion, '8.1.0');
```

Comment:
Sometimes it happens that the version we get from a server is malformed or missing and this will make this code throw breaking the whole tab, we should probably fallback to false in this case

Reply:
I forgot about this, it is an easy pitfall I could see us running into in the future (and has burned us before), we could have a separate shared util for doing this version check, maybe in `compass-utils`? I can create a ticket if that's something we would want.

Reply:
agreed; would be good to abstract away the semver dependency in general as well